CONTRIBUTING.md

```
@@ -128,3 +128,109 @@ for (auto const& listEntry : *list) { /*...*/ } // avoid
+  for the release
+
+  Consider moving still open items to another milestone
+
+## Check the CI Coverage
+
+* Remind/ask myself what platforms the release is targeting (I roughly
+  target the last two years of releases for ubuntu/fedora, debian
+  stable/oldstable + anything older, if relevant in GSI still)
+* Then I compare what is currently tested via CI and decide whether it
+  needs an updates, e.g.
+   * regenerate/update existing images (cover new FairSoft releases? get
```

Markdownlint

Expected: 2; Actual: 3
Unordered list indentation

(MD007, ul-indent)